pandas/tests/io/parser/test_common.py

```diff
@@ -63,13 +63,14 @@ def _set_noconvert_columns(self):
         "parse_dates": parse_dates,
         "delimiter": ",",
     }
+    parser.engine = "c"
```

Comment:
pls don't do this, rather pass an appropriate fixture

Reply:
@jreback I don't know how to pass in a fixture for this test since this test appears to define its own cparserwrapper. Ideally, this test should probably be moved to test_c_parser_only, and modified to use the c parser fixture. I feel like defining engine probably would be the best option for this PR as it maintains consistency with all the other engine fixtures.